model.py

```diff
@@ -116,12 +117,25 @@ def get_phi(self, args):
                 nn.ReLU())
             output_size = 576
 
-        return phi, output_size
+        elif args.architecture == 'pretrained':
+            assert args.phi_net_path is not None, 'Must specify path to feature network'
+            # Always load tensors onto CPU by default, will shift to GPU if necessary
+            phi, output_size = torch.load(args.phi_net_path)
```

Comment:
This comment is not what the subsequent line actually does.

Reply:
If the phi net is stored on the gpu and then saved, I think the load will try to load onto GPU and fail. We might want `map_location = cpu`